pcsx2/Frontend/GameList.cpp

```diff
@@ -224,12 +227,66 @@ bool GameList::GetIsoListEntry(const std::string& path, GameList::Entry* entry)
 	{
 		entry->title = std::move(db_entry->name);
 		entry->compatibility_rating = db_entry->compat;
-		if (StringUtil::StartsWith(db_entry->region, "NTSC-J"))
+							////// NTSC //////
```

Comment:
clang-format will probably realign this to the start next time the file is formatted.